global.json

```diff
@@ -1,22 +1,24 @@
 {
   "sdk": {
-    "version": "7.0.203",
+    "version": "8.0.100-preview.3.23178.7",
     "allowPrerelease": true,
     "rollForward": "latestPatch"
   },
   "tools": {
-    "dotnet": "7.0.203",
+    "dotnet": "8.0.100-preview.3.23178.7",
     "vs": {
       "version": "17.5",
```

Comment:
Update to VS version as well not needed here?

Reply:
No, not needed for now